### DISCUSSION TITLE
v2.0.0 Technical Preview ⑨

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2404] [分享 / Share]
**功能:**

- 组件与 `v1.12.20` 同步更新.
- 自定义顶栏:
  - 新增`新标签页打开`选项
  - 改善了拖动排列顶栏元素顺序时的性能.
  - `收藏`弹窗中修复了一些搜索相关的 bug.
  - 暂时禁止了`动态`上定时刷新的数字提醒, 因为弹窗里还没做完实时刷新. (#2303)
- 番剧区:
  - 修复宽屏模式 + mini 播放器出现的布局错乱. (PR #2371 by [FoundTheWOUT](https://github.com/FoundTheWOUT))
  - 恢复`默认播放器模式`, `启用双击控制`和`控制栏触摸优化`的支持.
- 夜间模式更新. (PR #2296 by [FoundTheWOUT](https://github.com/FoundTheWOUT))
- 修复 Firefox 下无法加载. (#2308)
- 下载视频中支持仅下载音频. (#2325)
- 更新了直播间勋章的 API.
- 修复夜间模式下部分 iframe 弹窗背景没有变透明.
- 修复组件没超过更新间隔期时, 选项里点检查更新不执行.

**开发:**

- 修复 `VSlider` 上使用方向键没有阻止默认行为.
- 修复 `MiniToast` 加载过早时获取不到 `body`.
- `addControlBarButton` 支持番剧区.

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.0.0-tp9'>v2.0.0 Technical Preview ⑨</a>.</em>

**Comment by SekiBetu** [comment]
![image](https://user-images.githubusercontent.com/38283893/137945017-1a55fe27-60a2-4f3d-8b3c-33ae0b99d470.png)
运行正常，不错，坐等下次预览版了

**Reply by SekiBetu** [reply]
现在原来分 `样式加载模式` 和 `功能加载模式` 了

![image](https://user-images.githubusercontent.com/38283893/137953948-04972cf8-a34b-4b26-bfa2-bf6cc245cbc3.png)

我说刚才怎么 `默认播放器模式` 功能没用
把 `功能加载模式` 设置成 `延后` 就好了，建议脚本的 `功能加载模式` 默认设置成 `延后`

![image](https://user-images.githubusercontent.com/38283893/137953846-06ed99f7-d5c0-4a44-88c4-3072e2138304.png)

**Reply by the1812** [reply]
功能起不起作用不应该跟加载模式有关, 这里可能还是有 bug

**Comment by Geeyun-JY3** [comment]
用这个「⑨」不利于检索啊（看到新 Releases 能搜索有感而发

**Reply by the1812** [reply]
皮一下(

**Reply by high0** [reply]
用这个⑨容易和神社搞混是吧23333

**Comment by ACE-MiRacle** [comment]
请问我的edge为什么一直显示这个安装不了？
![image](https://user-images.githubusercontent.com/65381143/139426492-ebb2d2c8-5c4d-4732-8ac3-26c74df08c0b.png)

**Reply by the1812** [reply]
只有 github 源能用, 装完设置里面更新源也要换 github
